spec/build/script/node_js_spec.rb

```diff
@@ -32,6 +32,32 @@
         should include_sexp [:cmd, 'nvm install 0.9', echo: true, timing: true]
       end
 
+      context 'add cache by default' do
+        it 'adds ${TRAVIS_HOME}/.cache/node_modules to directory cache' do
+          should include_sexp [:cmd, 'rvm $(travis_internal_ruby) --fuzzy do $CASHER_DIR/bin/casher add ${TRAVIS_HOME}/.cache/node_modules', timing: true]
+        end
+      end
+
+      context 'when cache key is not specified' do
+        it 'adds ${TRAVIS_HOME}/.cache/node_modules to directory cache' do
+          should include_sexp [:cmd, 'rvm $(travis_internal_ruby) --fuzzy do $CASHER_DIR/bin/casher add ${TRAVIS_HOME}/.cache/node_modules', timing: true]
+        end
+      end
+
+      context 'when cache is explicitly set' do
+        let(:config) { { cache: { npm: {} } } }
```

Comment:
I don't think this is something we'd see in production (since it correspond to this YAML snippet:

```yaml
cache:
  npm: {}
```

Perhaps you meant:

```
let(:config) { { cache: "npm" } }
```